src/quo2/components/list_item.cljs

```diff
@@ -0,0 +1,250 @@
+                             ;; The number 80px was determined by trial and error.
+                             :flex-basis      80}
+                            accessory-style)}
+     [rn/view {:style (:tiny spacing/padding-horizontal)}
+      (case accessory
+        :radio    [controls/radio {:value active :disabled disabled}]
+        :checkbox [(if animated-accessory?
+                     controls/animated-checkbox
+                     controls/checkbox)
+                   {:value active :disabled disabled}]
+        :switch   [controls/switch {:value active :disabled disabled}]
+        :text     [text/text {:color           :secondary
+                              :ellipsize-mode  :middle
+                              :number-of-lines 1}
+                   accessory-text]
+        accessory)]
+     (when (and chevron platform/ios?)
+       [rn/view {:style {:padding-right (:tiny spacing/spacing)}}
+        [icons/icon :main-icons/next {:container-style {:opacity         0.4
+                                                        :align-items     :center
+                                                        :justify-content :center}
+                                      :resize-mode     :center
+                                      :color           (theme-colors colors/neutral-50 colors/neutral-50)}]])]))
+
+(defn list-item
+  [{:keys [type accessory disabled subtitle-max-lines icon icon-container-style
+           left-side-alignment icon-color
+           title subtitle subtitle-secondary active on-press on-long-press chevron size text-size
+           accessory-text accessibility-label title-accessibility-label accessory-style
+           haptic-feedback haptic-type error animated animated-accessory? title-text-weight container-style
+           active-background-enabled]
+    :or   {subtitle-max-lines        1
+           type                      :main
+           haptic-feedback           true
+           animated                  platform/ios?
+           active-background-enabled true
+           haptic-type               :selection}}]
+  (let [type           (if disabled :disabled type)
+        {:keys [text-color active-background passive-background]}
+        (themes type)
+        icon-color      (or icon-color (:icon-color (themes type)))
+        optional-haptic (fn []
+                          (when haptic-feedback
+                            (haptic/trigger haptic-type)))
+        component       (cond
+                          (and (not on-press)
+                               (not on-long-press))
+                          rn/view
+                          animated animated/pressable
+                          :else    gh/touchable-highlight)]
+    [rn/view {:background-color (if (and (= accessory :radio) active)
+                                  active-background
+                                  passive-background)}
+     [component
```

Comment:
probably there is a better name ? component is too general

Reply:
changed to `touchable-view`